CHANGES.yaml

```diff
@@ -1,3 +1,23 @@
+- 2.1.0:
+    - Added Juju 2.0-beta support.
+    - Updated all api calls to support Juju 1.x and 2.x-beta facades.
+    - Added the ability to create and switch between models in Juju 2.0.
+    - Updated terminology to match the Juju 2.0 terminology.
+    - >
+      Created user profile view which shows you your available models, bundles
+      and charms after logging into the charmstore.
+    - >
+      Added support for syntax highlighting in the charm details pages in the
+      charmbrowser when the charm author provides a Github Flavoured Markdown
+      readme file.
+    - >
+      Added the ability to drag uncommitted units between machines in the
+      machine view.
+    - Unit statuses are now also shown in the machine view.
+    - Many UI tweaks and updates.
+    - (FIX) When subordinates are deployed extra empty machines are created.
+    - (FIX) Websockets weren't closed properly when switching models.
```

Comment:
Your tenses between these FIX items don't match.  I think they should be written from the perspective of the applied fix, i.e.

- When subordinates are deployed extra empty machines are not created.
- Websockets are now closed properly when switching models.
- When a user logs out all cookies are deleted.